GIT_CONFLICTS.md

```diff
@@ -0,0 +1,98 @@
+## Common QnA
+
+<details>
+<summary>
+<em>How can I know if I’m in a bad situation?</em>
+</summary>

```

Comment:
I meant for all the things, So kindly consider the review comments for  the entire file, that would be highly appreciated.
We expect you understand and then come up with good refactor, instead of just doing it for the sake of it